lib/Conversion/TorchToStablehlo/Basic.cpp

```diff
@@ -2091,6 +2091,206 @@ LogicalResult ConvertAtenOp<AtenTrilOp>::matchAndRewrite(
   return success();
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenScaledDotProductAttentionOp>::matchAndRewrite(
```

Comment:
How about writing it as a decomposition? So it could be reused by linalg/tosa backend?

Reply:
Sounds a good idea~,
 I will check how to implement it